### ISSUE TITLE
Can't view XML

### ISSUE THREAD

**Issue by rcourtna** [gvalkov/tailon-legacy#1]
Hi,
I'm using Tailon to look at a file that logs the XML responses from an API. The XML is not viewable in Tailon. Guessing some encoding needs to happen somewhere?

tailon, version 0.2.1 (installed via pip)

**Comment by gvalkov** [comment]
Hello,

Tailon is not very sophisticated with regard to encodings at the moment (everything is decoded as utf8). I will try to address this in a future version. Could you perhaps attach one of these xml files?

Regards,
Georgi

**Comment by rcourtna** [comment]
Hi,

I have no problem with the UTF8 character encodings. It's just characters that a web browser will interpret rather than show. Things like greater than ">" and less than "<" should be converted to html entities before display "&gt;", "&lt;".

Save the following into a text file and look at it with Tailon:

```
+++ START POST Mar 08 2014 9:01:27 +++
POST URL: http://192.168.2.98:8080/Adapter.XXX/1R6TFBY588/UpdateUser

POST DATA: address-city=&address-country=&address-postcode=&address-state=&address-street=&email=us345%40benevity.invalid&firstname=&hmac=8ASud1773DAiWuRZylHSkY7EstU%3D&initials=&lastname=&user=13-531b3dab3b09f

SERVER RESPONSE FOLLOWS:
<?xml version="1.0" encoding="UTF-8" ?>
<response type="UpdateUser" status="SUCCESS">
  <request>
    <param name="address-state" value=""/>
    <param name="address-postcode" value=""/>
    <param name="address-city" value=""/>
    <param name="initials" value=""/>
    <param name="hmac" value="8ASud1773DAiWuRZylHSkY7EstU="/>
    <param name="email" value="us345@benevity.invalid"/>
    <param name="address-street" value=""/>
    <param name="address-country" value=""/>
    <param name="lastname" value=""/>
    <param name="firstname" value=""/>
    <param name="user" value="13-531b3dab3b09f"/>
  </request>
  <messages/>
<content type="text/xml; charset=utf-8">
  <user id="4N9Z6FAPYD"/>
</content>
</response>
+++ END POST Mar 08 2014 9:01:27 +++
```

Thanks!

**Comment by gvalkov** [comment]
Oops. Fixed in e7e2a1221ec7bf48990e50ad5f2f8e00f564c548.  Feel free to upgrade to tailon 0.2.2. 

Thanks for the catch!

**Comment by rcourtna** [comment]
thanks!  Any idea how long it takes for this to get into the PIP repository?

EDIT: Scratch that. Chrome was using a cached version of the Tailon javascript. Shift-refresh and I'm in business. Thanks!
